fiat-roles/src/main/java/com/netflix/spinnaker/fiat/permissions/RedisPermissionsRepository.java

```diff
@@ -108,6 +109,7 @@ public RedisPermissionsRepository put(@NonNull UserPermission permission) {
 
       String userId = permission.getId();
       insertNewValuesPipeline.sadd(allUsersKey(), userId);
+      insertNewValuesPipeline.hset(userId, adminKey(), String.valueOf(permission.isAdmin()));
```

Comment:
Instead of a hash with the key userId with just a single entry adminKey -> "true" or "false" how about using a set with the key adminKey that just contains the users that are admins

means just one key to deal with, it means that bulk operations on admins are a bit easier. the save user just becomes a sadd or srem and the test for user being admin is sismember all of which are O(1)